src/components/Auth/SignUpForm.js

```diff
@@ -0,0 +1,107 @@
+import ResponsiveText from "@components/ResponsiveText";
+import {
+  Button,
+  ButtonGroup,
+  Divider,
+  IconButton,
+  Stack,
+  TextField,
+  Typography,
+  useTheme,
+} from "@mui/material";
+import React from "react";
+import GoogleIcon from "@mui/icons-material/Google";
+import TwitterIcon from "@mui/icons-material/Twitter";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import RoundedButton from "@components/buttons/RoundedButton";
+
+const SignUpForm = () => {
+  const theme = useTheme();
+
+  const SocialButton = ({ color, children }) => (
+    <IconButton
+      size="large"
+      sx={{
+        backgroundColor: color,
+        color: theme.palette.primary.white,
+        "&:hover": {
+          backgroundColor: color,
+        },
+      }}
+    >
+      {children}
+    </IconButton>
+  );
+
+  return (
+    <Stack
+      alignItems="center"
+      justifyContent="center"
+      spacing={3}
+      sx={{
+        height: "100%",
+      }}
+    >
+      <ResponsiveText variant="h2" sx={{ mb: 2 }}>
+        Create An Account
+      </ResponsiveText>
+
+      <Typography>Join Maker Academy as a</Typography>
+
+      <ButtonGroup variant="outlined" fullWidth sx={{ maxWidth: 450 }}>
```

Comment:
Change from MUI ButtonGroup to [MUI Toggle Button](https://mui.com/components/toggle-button/#exclusive-selection) (exclusive selection variant).